pkg/kubectl/cmd/config/view.go

```diff
@@ -99,7 +99,10 @@ func NewCmdConfigView(f cmdutil.Factory, streams genericclioptions.IOStreams, Co
 	return cmd
 }
 
-func (o *ViewOptions) Complete(cmd *cobra.Command) error {
+func (o *ViewOptions) Complete(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
+		return cmdutil.UsageErrorf(cmd, "unexpected args: %v", args)
```

Comment:
We should probably say use full "arguments" here, the same applies to `pkg/kubectl/cmd/apiresources/apiversions.go` and `pkg/kubectl/cmd/apiresources/apiresources.go` would you mind fixing the other two as well?

Reply:
Thanks! 
Updated, please take a look.